limitador/src/storage/redis/redis_cached.rs

```diff
@@ -238,10 +238,11 @@ impl CachedRedisStorage {
         let mut interval = tokio::time::interval(flushing_period);
         tokio::spawn(async move {
             loop {
-                if p.load(Ordering::Acquire) {
-                    if storage.is_alive().await {
-                        warn!("Partition to Redis resolved!");
-                        p.store(false, Ordering::Release);
+                let partionned = !storage.is_alive().await;
```

Comment:
I wonder if it'd be better renaming `partitionned` as `notAlive` or something, only because there's `partitioned`, `p`, `fn partitioned`, and now `partitionned`

I'm not tied to this though but the term feels a bit overloaded

Reply:
Sure I just think it's hard to read with so many instances of `partitioned`. As I've been looking at it recently it's not too difficult because I know what each variant is but if I had come to this fresh it'd be easy to miss the additional `n` being something different.

Reply:
Oh I already misread it, it's `partionnned` not even `partitionned`

Reply:
Duh! Yes... 🤣

Reply:
Fixed the both types